bin/bundle-accp-jdk8.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+set -x
+
+# Use this script for JDKs 8. Please note that this script does not modify the security file,
+# so one still needs to programmatically install ACCP.
+#
+# This script requires JAVA_HOME set properly.
+#
+# usage: ./bundle-script 2.3.3 linux-x86_64
+
+ACCP_VERSION=$1 # for example 2.3.3
+ACCP_PLATFORM=$2 # for example linux-x86_64 or osx-aarch64
+FIPS_EXTENSION=${3:""} # for FIPS, pass -FIPS
+BUILD_DIR="$(pwd)/build"
+TMP_DIR="$BUILD_DIR/private/tmp"
+ACCP_PATH="$TMP_DIR/accp"
+JDK_PATH="$JAVA_HOME"
+JAVA_BIN="$JDK_PATH/bin"
+
+function download_accp {
+    wget -O "$ACCP_PATH/accp.jar" "https://repo1.maven.org/maven2/software/amazon/cryptools/AmazonCorrettoCryptoProvider${FIPS_EXTENSION}/${ACCP_VERSION}/AmazonCorrettoCryptoProvider${FIPS_EXTENSION}-${ACCP_VERSION}-${ACCP_PLATFORM}.jar"
+}
+
+function mk_all_dirs {
+    mkdir -p "$ACCP_PATH"
```

Comment:
In the JDK 11+ script you run `clean_all_dirs` inside `mk_all_dirs`, did you want to do the same for this JDK8 script?

Reply:
Thanks for catching this. Done.